scripts/settings.js

```diff
@@ -36,11 +35,7 @@ function getDefaultCurrency() {
     const langCode = navigator.languages[0]?.split('-')?.at(-1) || 'US';
     return getCurrencyByAlpha2(langCode)?.toLowerCase() || 'usd';
 }
-/**
- * The user-selected Price Oracle, used for all price data
- * @type {Oracle}
- */
-export let cOracle = new Oracle();
+
```

Comment:
If we're removing the Oracle instantiation from here, we should at least add some method or settings variable for us to "change" between instances in real-time via Settings in the future, as this is not intended to be a hardcoded single class, it should function similar to `cExplorer` or `cNode`, which are both stored in `settings.js`.

Reply:
Having a setting to choose where you want to get the price from maybe is a bit too much...
Regardless we can do this change when we will actually implement the feature?

Reply:
I think users should have full control over their data 🤷‍♂️ 

And sure, we can save it for when more are hosted.